### PR TITLE
CRUD

### PR DESCRIPTION
**Pull request by Kamba56**
-Added Records for Users. posts, comments and likes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,52 @@
...
-* Database initialization
+### Setup
+- clone the repo using
+    ```git clone https://github.com/Kamba56/School_Library.git```
+- Change directory
+    `cd School_Library`
 
-* How to run the test suite
+### Deployment
+Run `ruby main.rb`
```
Comment (FrederickMih) [inline README.md:24]: - [ ] Kindly update the content of these headers in the README to reflect your blog app. This is not a school library app.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,52 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (FrederickMih) [inline README.md:52]: - [ ] Kindly fix :wrench:  this Licensed link. It's broken

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @Kamba56, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

- Gitflow has been followed :+1: 
- Good commit history :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (peterrobert) [reply README.md:52]: This change has been done well

File: README.md (hunk 1 above)
Comment (peterrobert) [reply README.md]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @Kamba56 , 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**